Description Fix file encoding to prevent an error that would otherwise lead to empty doc packages.
Author: Markus Koschany <apo@debian.org>
Bug: https://bugs.debian.org/798343
--- a/ant/bc+-build.xml
+++ b/ant/bc+-build.xml
@@ -107,7 +107,8 @@
                     srcdir="${artifacts.dir}/@{target}/src"
                     destdir="${build.dir}/@{target}/classes"
                     memoryMaximumSize="512m"
-                    debug="${release.debug}" fork="true">
+                    debug="${release.debug}" fork="true"
+                    encoding="UTF-8">
                     <classpath>
                         <path refid="project.classpath" /> 
                         <fileset dir="${artifacts.jars.dir}">
@@ -144,7 +145,8 @@
                     srcdir="${artifacts.dir}/@{target}/src"
                     destdir="${build.dir}/@{target}/classes"
                     memoryMaximumSize="512m"
-                    debug="${release.debug}">
+                    debug="${release.debug}"
+                    encoding="UTF-8">
                     <classpath>
                         <path refid="project.classpath" /> 
                         <fileset dir="${artifacts.jars.dir}">
@@ -222,7 +224,8 @@
             <javadoc maxmemory="384m" sourcepath="@{srcDir}"
                      destdir="@{docsDir}"
                      windowtitle="Bouncy Castle Library ${release.name} API Specification"
-                     header="&lt;b&gt;Bouncy Castle Cryptography Library ${release.name}&lt;/b&gt;">
+                     header="&lt;b&gt;Bouncy Castle Cryptography Library ${release.name}&lt;/b&gt;"
+                     encoding="UTF-8">
                 <docElements/>
                 <arg value="${javadoc.args}" />
                 <classpath>
@@ -288,7 +291,8 @@
             destdir="${lcrypto.target.classes.dir}"
 	    memoryMaximumSize="512m"
             fork="true"
-            debug="${release.debug}">
+            debug="${release.debug}"
+            encoding="UTF-8">
             <classpath>
                 <path refid="project.classpath" /> 
             </classpath>
